File: app/src/main/java/com/nerdstone/neatform/NeatFormApp.java
package com.nerdstone.neatform;

import android.app.Application;
import timber.log.Timber;
import timber.log.Timber.DebugTree;

public class NeatFormApp extends Application {

  @Override
  public void onCreate() {
    super.onCreate();
    if(BuildConfig.DEBUG) {
      Timber.plant(new DebugTree());
    }
  }
}

File: app/src/main/java/com/nerdstone/neatform/NeatFormApp.kt
package com.nerdstone.neatform

import android.app.Application
import timber.log.Timber
import timber.log.Timber.DebugTree

class NeatFormApp : Application() {

    companion object {
        lateinit var instance: Application
            private set
    }

    override fun onCreate() {
        super.onCreate()
        if (BuildConfig.DEBUG) {
            Timber.plant(DebugTree())
        }
        instance = this
    }
}
